src/OrleansServiceFabricUtils/project.json

```diff
@@ -0,0 +1,12 @@
+﻿{
+  "dependencies": {
+    "Microsoft.ServiceFabric.Services": "2.1.163",
+    "Newtonsoft.Json": "7.0.1"
```

Comment:
Note that this is no longer a transitive dependency since it's being used to return the endpoints to SF